guides/doc-Upgrading_and_Updating/topics/updating_satellite_server_to_next_minor_version.adoc

```diff
@@ -70,120 +60,209 @@ If you lose connection to the command shell where the upgrade command is running
+baseurl=https://satellite.example.com/pulp/content/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/satellite/6.10/os/
 enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
 
 [{RepoRHEL7ServerSatelliteMaintenanceProductVersion}]
 name={ProjectName} Maintenance 6 for RHEL 7 Server RPMs x86_64
-baseurl=file:///var/lib/pulp/published/yum/https/repos/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/sat-maintenance/6/os/
+baseurl=https://satellite.example.com/pulp/content/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/sat-maintenance/6/os/
 enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+sslverify = 1
 
 [{RepoRHEL7ServerSoftwareCollections}]
 name=Red Hat Software Collections RPMs for Red Hat Enterprise Linux 7 Server x86_64
-baseurl=file:///var/lib/pulp/published/yum/https/repos/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/rhscl/1/os/
+baseurl=https://satellite.example.com/pulp/content/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/rhscl/1/os/
 enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+sslverify = 1
 ----
 +
-. In the configuration file, replace `My_Organization` in the `baseurl` with the correct organization label.
-To obtain the organization label, enter the command:
+* In the configuration file, replace 
+** `/etc/pki/katello/certs/org-debug-cert.pem` in `sslclientcert` and `sslclientkey` with the location of the downloaded organization debug certificate.
+** `satellite.example.com` with correct FQDN 
+** `My_Organization` with the correct organization label in the `baseurl`. To obtain the organization label, enter the command:
++
+----
+# hammer organization list
+----
+
+* Run the `reposync` command as follows:
 +
+[options="nowrap" subs="attributes"]
 ----
-# ls /var/lib/pulp/published/yum/https/repos/
+# cd ~
+# reposync --delete --download-metadata -p satellite_repos -n \
+ -r {RepoRHEL7ServerAnsible} \
+ -r {RepoRHEL7Server} \
+ -r {RepoRHEL7ServerSatelliteServerProductVersion} \
+ -r {RepoRHEL7ServerSatelliteMaintenanceProductVersion} \
+ -r {RepoRHEL7ServerSoftwareCollections}
 ----
+
+* This will download the contents of these repositories from the Upstream Satellite and store it in the directory `~/satellite-repos`.
+* `reposync` in RHEL 7 at this point does not download the yum metadata. It downloads just the rpms. Due to we would need to manually run `createrepo .` in each sub-directory of `satellite-repos`. You may use the following to automate this. 
```

Comment:
```suggestion
The `reposync` command in {RHEL} 7 does not download the RPMs but not the Yum metadata. 
Because of this, you must manually run `createrepo .` in each sub-directory of `{Project}-repos`. 
You can use the following command to automate this: 
```